tests/testsuites/sndrcv_relp_tls_rcvr.conf

```diff
@@ -2,7 +2,7 @@ $IncludeConfig diag-common.conf
 
 module(load="../plugins/imrelp/.libs/imrelp")
 # then SENDER sends to this port (not tcpflood!)
-input(type="imrelp" port="13515" tls="on")
+input(type="imrelp" port="13515" tls="on" tls.prioritystring="SYSTEM")
```

Comment:
This should be done in a new test (copy of this one) because otherwise we do not test the default case, which seems equally important.